packages/fether-react/src/Accounts/AccountsList/AccountsList.js

```diff
@@ -46,58 +47,60 @@ class AccountsList extends Component {
     const accountsListLength = accountsList && accountsList.length;
 
     return (
-      <div className='accounts-list'>
-        <Header
-          right={
-            <Clickable
-              className='icon -new'
-              onClick={this.handleCreateAccount}
-            />
-          }
-          title={<h1>Accounts</h1>}
-        />
+      <RequireHealthOverlay require='node'>
```

Comment:
Instead of adding `<RequireHealthOverlay require='node'>` to each child route, could you actually add it in the parent `src/Accounts/Accounts.js` component?

Reply:
I've previously tried wrapping Accounts.js and it didn't work
I've also tried wrapping CreateAccount.js and it didn't work.
I just tried wrapping around just the routes in CreateAccount.js, but that also doesn't work, i.e.
```
              <RequireHealthOverlay require='node'>
                <Route
                  component={StepComponent}
                  key={`Step${index + 1}`}
                  path={`/accounts/new/${index + 1}`}
                />
              </RequireHealthOverlay>
```
So I think it's a React Router issue that's making us have to settle for wrapping each individual component that is a route where we want the overlay applied.

Reply:
Ah ok, i wasn't aware of that.